kolibri/plugins/learn/assets/src/views/LibraryPage/PinnedNetworkResources.vue

```diff
@@ -107,9 +109,10 @@
 
     position: relative;
     display: inline-flex;
-    width: 100%;
-    max-height: 258px;
+    width: 350px;
```

Comment:
I think you should be able to remove all of the styles on this page, as they seem to be only used for the `Explore` card? If so, anything for that care can be defined in that file. It is redundant and makes it a little bit confusing to debug, and things might be overwriting each other in confusing ways.